example-app/src/services/NasaService.ts

```diff
@@ -0,0 +1,35 @@
+enum PatentData {
+  Id,
+  Title = 2,
+  Description,
+  ImageUrl = 10
+}
+
+export type PatentDataShape = {
+  id?: string,
+  title: string,
+  description: string,
+  imageUrl: string
+}
+
+export class NasaService {
```

Comment:
instantiate this service right here (below)

and export only instance instead of service.